### PR TITLE
User Reservation Styling

### PR DESCRIPTION
**Pull request by clee4037**
Summary:
- Finished creating layout for each sub component
- Refactored search and map view components

Next: 
- Final css/styling touches
- Test edge cases
- Modularize code -- search bar and map sidebar
- Readme

### CODE REVIEW

File: src/components/user-reservation/Confirmation.jsx [hunk 1]
```diff
@@ -11,21 +37,43 @@ const Confirmation = () => {
     city: "San Francisco",
     zip: "94131",
     state: "CA",
+    phoneNumber: "123-456-7890",
     username: "Bruce",
```
Comment (bzw24) [inline src/components/user-reservation/Confirmation.jsx]: We going to put this into existence, so I can own a Tesla in the future

File: src/components/user-reservation/SearchBar.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  fetchClosestGarages,
+  setPage,
+  setSearch,
+} from "../../utils/slice/reservationSlice";
+
+const SearchBar = () => {
+  const { reservation, search } = useSelector((state) => state.reservation);
+  const dispatch = useDispatch();
+
+  const handleSubmit = async (e) => {
```
Comment (bzw24) [inline src/components/user-reservation/SearchBar.jsx]: Good use of providing client side checks prior to submission

File: package.json
Comment (bzw24) [inline package.json:1]: wait I thought we are using heroicons? Don't think it matters though, so use whatever you need for the styling if heroicons does not have what you need.

**Review by bzw24 — APPROVED** [review]
Okay with styling, comments provided

File: package.json
Comment (clee4037) [reply package.json:1]: I'll update

**Review by stevkim — APPROVED** [review]
Nice dude! Your redux code looks awesome
